Server/src/controllers/getDataController.ts

```diff
@@ -0,0 +1,11 @@
+const getDataService = require('../services/getDataService');
```

Comment:
This is all still following javascript structure not the new TS structure. Needs to be fixed - see Refactoring to TS

Reply:
Should be fixed now